### PR TITLE
If no command is given show help text; and instructions on sub-help. Include a dev quick start in the README

### PR DESCRIPTION
**Pull request by cleve-fauna**
Ticket(s): FE-6153

## Problem

If no command is given we are showing some generic red text.

Screenshot 2024-11-22 at 3 37 18 PM

## Solution

Instead just show help, and how to get more detailed help
Screenshot 2024-11-22 at 4 20 02 PM

## Result

No weird errors shown.

## Out of scope

We should do the same thing for missing sub commands.

Tracking here: https://faunadb.atlassian.net/browse/FE-6154

## Testing

Ran the suite.

### CODE REVIEW

**Review by ecooper — APPROVED** [review]

File: DEV-README.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+### Quick Start
+
+```
+npm install
+npm run build
+./src/user-entrypoint.mjs
```
[the remark this answers is not in the document]
Comment (echo-bravo-yahoo) [reply DEV-README.md:6]: Tests require builds (b/c we assert things on the behavior of the prod bundle), so I think it's reasonable to include?

File: test/general-cli.mjs [hunk 2]
```diff
@@ -40,7 +40,7 @@ describe("cli operations", function () {
   });
 
   // TODO: this doesn't work because turning on strict mode breaks parsing sub-commands. why?
-  it("should exit with a helpful message if a non-existant command is provided", async function () {
+  it("should exit with a helpful message if a non-existent command is provided", async function () {
```
Comment (echo-bravo-yahoo) [inline test/general-cli.mjs:43]: oopsie

**Review by echo-bravo-yahoo — APPROVED** [review]

File: DEV-README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (cleve-fauna) [reply DEV-README.md:6]: i left it in to make sure your stuff wasn't roont. 

Make sense?

Cause my stuff was roont.

[the remark this answers is not in the document]
Comment (cleve-fauna) [reply DEV-README.md:6]: i'll add some comments so noone else brain freezes when they read it.

**Pull request merged**